### ISSUE TITLE
 Issue 2: Image Alt Text Missing

### ISSUE THREAD

**Issue by Jessicaarrora** [Jessicaarrora/Assignment-2#2]
Description: Some images on the “Projects” page lack descriptive alt text.
Details: Certain images do not have any alt text, affecting accessibility and SEO.
Steps to Reproduce:
Navigate to the “Projects” page.
Inspect images for alt text attributes.
Expected Result: All images should have descriptive alt text.
Actual Result: Several images are missing alt text.
Resolution: Add appropriate alt text to all images on the page.

**Comment by Jessicaarrora** [comment]
before: 
<img width="1056" alt="Screenshot 2024-07-14 at 7 41 15 PM" src="https://github.com/user-attachments/assets/fdad1565-5a6b-4f7b-9798-24d96e33d776">

after: 
<img width="1020" alt="Screenshot 2024-07-16 at 6 48 27 PM" src="https://github.com/user-attachments/assets/2ae8657b-7df4-4338-a0b3-560a30a14f44">
